Deck.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+class Deck
+{
+
+    // Properties
+    private int $id;
+    private string $name;
+    private string $format;
+    private string $colourID;
+    private string $archetype;
+    private string $last_updated;
+    private string $primer;
+    private string $image;
+    private string $moxfield_link;
+
+
+    public function __construct(
+        string $name, string $format, string $colourID, string $archetype, string $last_updated
+        , string $primer, string $image, string $moxfield_link, int $id = -1
+    )
+    {
+        $this->id = $id;
+        $this->name = $name;
+        $this->format = $format;
+        $this->colourID = $colourID;
+        $this->archetype = $archetype;
+        $this->last_updated = $last_updated;
+        $this->primer = $primer;
+        $this->image = $image;
+        $this->moxfield_link = $moxfield_link;
+
+    }
+
+    /**
+     * @return int
+     */
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    /**
+     * @param int $id
+     */
+    public function setId(int $id): void
+    {
+        $this->id = $id;
+    }
```

Comment:
Do you actually need these setters at this point? If not, get em in the bin!

Reply:
Removed Getters and Setters for ID - but left some of the others for Story 2